Constitution.txt

```diff
@@ -1,490 +1,264 @@
-
-3: The Senators and Representatives before mentioned, and the Members of the several State Legislatures, and all executive and judicial Officers, both of the United States and of the several States, shall be bound by Oath or Affirmation, to support this Constitution; but no religious Test shall ever be required as a Qualification to any Office or public Trust under the United States.
-
-Article VII (Article 7 - Ratification)
-The Ratification of the Conventions of nine States, shall be sufficient for the Establishment of this Constitution between the States so ratifying the Same.
-
-The Word "the", being interlined between the seventh and eight Lines of the first Page, The Word "Thirty" being partly written on an Erazure in the fifteenth Line of the first Page. The Words "is tried" being interlined between the thirty second and thirty third Lines of the first Page and the Word "the" being interlined between the forty third and forty fourth Lines of the second Page.
-
-  	
-done in Convention by the Unanimous Consent of the States present the Seventeenth Day of September in the Year of our Lord one thousand seven hundred and Eighty seven and of the Independence of the United States of America the Twelfth In witness whereof We have hereunto subscribed our Names,
-
-Attest William Jackson Secretary	
-  Go: Washington -Presidt. and deputy from Virginia Showing George Washington's signature.
-
-
-Delaware
-  Geo: Read
-  Gunning Bedford jun
-  John Dickinson 
-  Richard Bassett 
-  Jaco: Broom
-
-
-Maryland
-  James McHenry
-  Dan of St   Thos. Jenifer 
-  Danl Carroll.
-
-
-Virginia
-  John Blair—
-  James Madison Jr.
-
-
-North Carolina
-  Wm Blount
-  Richd. Dobbs Spaight.
-  Hu Williamson
-
-
-South Carolina
-  J. Rutledge
-  Charles Cotesworth Pinckney
-  Charles Pinckney
-  Pierce Butler.
-
-
-Georgia
-  William Few
-  Abr Baldwin
-
-
-New Hampshire
-  John Langdon
-  Nicholas Gilman
-
-
-Massachusetts
-  Nathaniel Gorham
-  Rufus King
-
-
-Connecticut
-  Wm.   Saml. Johnson
-  Roger Sherman
-
-
-New York
-  Alexander Hamilton
-
-
-New Jersey
-  Wil. Livingston
-  David Brearley.
-  Wm. Paterson. 
-  Jona: Dayton
-
-
-Pennsylvania
-  B Franklin
-  Thomas Mifflin
-  Robt Morris
-  Geo. Clymer
-  Thos. FitzSimons 
-  Jared Ingersoll 
-  James Wilson. 
-  Gouv Morris
-
-
-
-Letter of Transmittal
-skip to Letter of Transmittal to Congress   up to the Constitution
-In Convention. Monday September 17th 1787.
-Present
-The States of
-New Hampshire, Massachusetts, Connecticut, Mr. Hamilton from New York, New Jersey, Pennsylvania, Delaware, Maryland, Virginia, North Carolina, South Carolina and Georgia.
+1: The government of any State may, at its discretion and with the assent of its Citizens, according to the State's constitution, elect to leave the Union, and inform the President of these United States in writing, signed by the State's chief executive. 
```

Comment:
Section 4 now describes a formal process for Secession. As we discovered in 1861, the lack of these rules makes any secession event extremely likely to come to violence.

In creating this section, I draw lessons from that history, as well as observations of the process of Brexit and its many impacts on the British Isles and European Union.